src/styles.scss

```diff
@@ -234,12 +234,16 @@ $green-font-color: #19B028;
 .font-weight-normal { font-weight: 400 }
 // font-weight-bold: 700
 
+// Flex Gaps
+.flex-gap { gap: 4px }
 
 // Widths
 .w-10 { width: 10% }
 .w-15 { width: 15% }
 .w-20 { width: 20% }
+//why w-30 is 10%? cc @zordon @jacksondean
 .w-30 { width: 10% }
```

Comment:
This is an excellent question. Will explore what classes are currently using this, and change it to the correct width soon